pallets/ddc-staking/src/benchmarking.rs

```diff
@@ -105,4 +105,22 @@ benchmarks! {
 	verify {
 		assert!(Ledger::<T>::contains_key(&new_controller));
 	}
+
+	allow_cluster_manager {
```

Comment:
Do you have plans to extend the cluster manager logic? If so, having a separate pallet for that would be nice.

Reply:
Planned features are about staking for cluster managers which will be similar to existing roles (edge, storage). Looks like at least staking and participation-related logic could live here.